test/guardian/sweeper_test.exs

```diff
@@ -0,0 +1,24 @@
+defmodule Guardian.DB.Test.SweeperTest do
+  use Guardian.DB.Test.DataCase
+
+  alias Guardian.DB.Token
+  alias Guardian.DB.Token.Sweeper
+
+  test "purge stale tokens" do
+    Token.create(%{"jti" => "token1", "exp" => Guardian.timestamp() + 5000}, "Token 1")
+    Token.create(%{"jti" => "token2", "exp" => Guardian.timestamp() - 5000}, "Token 2")
+
+    interval = 0
+    state = %{interval: interval}
+    new_state = Sweeper.sweep(self(), state)
```

Comment:
@doomspork the power or moving out from the GenServer, I dont need to speed up a genserver ❤️